virttest/vt_console.py

```diff
@@ -43,6 +43,7 @@ class ConsoleManager(object):
 
     def __init__(self):
         self._console = None
+        self.status_test_command = None
```

Comment:
I assume that status_test_command can be one attribute of ConsoleSession, nothing with ConsoleManager

Reply:
Do you want to change `status_test_command` after the `ConsoleSession` was created?
If so, I think you should change it through `vm.serial_console`.
Please note on line 63 `self._console.set_status_test_command(status_test_command)`, you can set this command during the session creation.

Reply:
@chunfuwen , if no define status_test_command for ConsoleManager, we need to use statement like below to intialize it in __init__() definition of ConsoleSession:

     def __init__(self, manager):
        self.__manager = manager
        self.status_test_command = manager._console.status_test_command

_console looks like one private attribute for ConsoleManager, so accessing it directly outside of ConsoleManager looks ugly. So I define status_test_command for ConsoleManger in this case.

Reply:
@hereischen , vm.serial_login() call ConsoleManager.create_session() with status_test_command parameter, and latter call self._console.set_status_test_command() to set status_test_command.
So I think it works.

Reply:
@czhang0  what is the problem you want to solve?
It's not about the whether it  does work or not. It's the reason why you want do it this way.
You can set `status_test_command` by calling `vm.serial_login()` without this patch.

Reply:
There are some access like 'session.status_test_command' in avocado-vt codes.
For example, line 804 in virttest/utils_net.py:
     804: session.sendline(session.status_test_command)
For serial login, ConsoleSession has no status_test_command.
Without this patch, complains will emit like ' ConsoleSession has no attribute status_test_command'.
So it's reason I add it.
Thanks for your comment!